gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
What is your merge process? I don't understand how all these whitespace, newline and unrelated file change issues can happen.